templates/blog/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "blog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@grafbase/sdk": "~0.0.20"
```

Comment:
use the latest version?

Reply:
This was generated by the CLI.

Reply:
@pimeys do we want to set a specific version and updated the templates on each release? Sounds like a lot

Reply:
no we do `~0.1.0` now which matches every version until 1.0